### DISCUSSION TITLE
How to get an account for the translation demo

### DISCUSSION THREAD

**Discussion by hamaryns** [foodcoops/foodsoft#960] [Q&A] [answered]
Hi, the translation wiki refers to https://app.foodcoops.net/demo/login, but how to get an account for that?
Cheers, H.

**Comment by paroga** [comment, marked as answer]
You find the credentials on our homepage: https://foodcoops.net/demo/#main-version

**Comment by hamaryns** [comment]
Thanks, will edit the wiki to add that, then.

**Comment by hamaryns** [comment]
OK, and how to get this in-place translation thing working? See https://github.com/foodcoops/foodsoft/wiki/Translation
